docs/source/index.rst

```diff
@@ -36,6 +36,34 @@ user:
     print(kennethreitz.followers_count)
 
     followers = [str(f) for f in gh.followers('kennethreitz')]
+    
+
+    # create a repository
+    gh.create_repository(name='A Repo')
+    
+    # grab the repo
+    repos = list(gh.repositories())
+    my_repo = [x for x in repos if x.name == 'A Repo'][0] # grab match
```

Comment:
Would it be more efficient to instead show off `my_repo = gh.repository(gh.me(), 'A Repo')` ? For someone like me, who has hundreds of repositories, this example wouldn't be very fast or memory friendly.

Reply:
Yes, that works much better. I wasn't aware of that functionality, I'll make some edits soon!